.circleci/config.yml

```
@@ -37,9 +37,9 @@ execution_steps: &execution_steps
     - run: hostPath=$HOME/.esvu/bin/$hostPath npm run ci
 
 jobs:
-  "Test262: verify tools; build & lint tests (Python 2)":
+  "Test262: verify tools; build & lint tests (Python 3)":
```

We might as well drop the now-unnecessary refinement.
```suggestion
  "Test262: verify tools; build & lint tests":
```